Code/GraphMol/Wrap/Atom.cpp

```diff
@@ -360,7 +360,10 @@ struct atom_wrapper {
         .def("GetPropNames", &Atom::getPropList, (python::arg("self")),
```

Comment:
n.b. private and computed values are now exposed to the API.  They were hidden/not exposed before.